### DISCUSSION TITLE
pybricks - writing data to a file and graph data

### DISCUSSION THREAD

**Discussion by familynygood** [pybricks/support#1802] [General]
Does pybricks have features of writing drivebase data to a file and graph them? 
If yes, please instruct how to do it.

**Comment by laurensvalk** [comment]
You can do something like this, and copy the data to your favorite spreadsheet tool:

If you clear the output before you run it, you can right-click and choose _select all_ to conveniently copy everything. 

![image](https://github.com/user-attachments/assets/5be86707-99b2-464e-84cc-ffc2837a0e13)

**Reply by familynygood** [reply]
Can you please kindly post the python code instead?
